packages/clerk-js/src/ui/primitives/Input.tsx

```diff
@@ -76,12 +76,12 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, ref)
    * type="email" will not allow characters like this one "ö", instead remove type email and provide a pattern that accepts any character before the "@" symbol
    */
 
-  const typeProps =
-    type === 'email'
-      ? {
-          pattern: '^.*@[a-zA-Z0-9\\-]+\\.[a-zA-Z0-9\\-\\.]+$',
-        }
-      : { type };
+  let value = rest.value;
+  let typeProps: Record<string, unknown> = { type };
+  if (typeof value === 'string' && type === 'email') {
+    value = value.trim();
+    typeProps = { pattern: '^.*@[a-zA-Z0-9\\-]+\\.[a-zA-Z0-9\\-\\.]+$' };
+  }
```

Comment:
❓ 
1. Is it possible to have `type='email'` but not a value that is a string ?
2. I'd expect with the changes made in our `useFormControl` hook (below) that we wouldn't need to call  `value = value.trim();` manually here. Am i missing something ?

Reply:
I just followed typescript here

Reply:
+1 on the second question. Shouldn't there be a `transformer` in `packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx` for `emailAddress`?

Reply:
There is a default transformer applied for type email in `useFormControl`

Reply:
Yeah, looks good now